services/proxy/pkg/middleware/oidc_auth.go

```diff
@@ -136,6 +143,19 @@ func (m OIDCAuthenticator) extractExpiration(aClaims oidc.RegClaimsWithSID) time
 	return defaultExpiration
 }
 
+func verifyExpiresAt(claims map[string]interface{}, cmp time.Time, req bool) bool {
```

Comment:
What is the `req` parameter for?

Reply:
I copied the code from jwt. for us this is a dead code path. I'll remove it. Less to think about.